admin/spec/controllers/spree/admin/webhook_subscribers_controller_spec.rb

```
@@ -0,0 +1,126 @@
+require 'spec_helper'
+
+describe Spree::Admin::WebhooksSubscribersController, type: :controller do
```

RuboCop (1.73)

[convention] Spec path should end with `spree/admin/webhooks_subscribers_controller*_spec.rb`.

(RSpec/SpecFilePathFormat)